src/data/mail.asm

```diff
@@ -0,0 +1,345 @@
+MACRO end_mail
+	dw MAIL_TERMINATOR
+ENDM
+
+MACRO no_card
+	dw $0000
+ENDM
+
+Mail:
+	dw .MailEmpty
+	dw .MailBlackBox
+	dw .MailBillsPCOutput
+	dw .MailMailboxIntro
+	dw .MailDeckDiagnosis1
+	dw .MailDeckDiagnosis2
+	dw .MailDeckDiagnosis3
+	dw .MailDeckDiagnosis4
+	dw .MailAutoDeckMachine1
+	dw .MailAutoDeckMachine2
+	dw .MailAutoDeckMachine3
+	dw .MailAutoDeckMachine4
+	dw .MailAutoDeckMachine5
+	dw .MailChallengeMachine
+	dw .MailGRChallengeMachine
+	dw .MailGrandMasterCup
+	dw .MailRonaldsScoutGR1
+	dw .MailRonaldsScoutGR2
+	dw .MailRonaldsScoutGR3
+	dw .MailRonaldsScoutGR4
+	dw .MailRonaldsScoutMorino
+	dw .MailRonaldsScoutCatherine
+	dw .MailRonaldsScoutHidero
+	dw .MailRonaldsScoutKanoko
+	dw .MailRonaldsScoutKamiya
+	dw .MailRonaldsScoutMami
+	dw .MailRonaldsScoutColorlessAltar
+	dw .MailRonaldsScoutBiruritchi
+	dw .MailLinkDuel
+
+.MailEmpty
+	tx MailboxEmptySenderText
+	tx MailboxEmptySubjectText
+	end_mail
+
+.MailBlackBox
+	tx MailBlackBoxOutputSenderText
+	tx MailBlackBoxOutputSubjectText
+	tx MailBlackBoxOutputBody1Text
+	db $00, MAIL_COMMAND_GIVE_BLACK_BOX
+	tx MailBlackBoxOutputBody2Text
+	no_card
+	end_mail
+
+.MailBillsPCOutput
+	tx MailBillsPCOutputSenderText
+	tx MailBillsPCOutputSubjectText
+	tx MailBillsPCOutputBody1Text
+	db $00, MAIL_COMMAND_GIVE_BILLS_PC
+	tx MailBillsPCOutputBody2Text
+	no_card
+	end_mail
+
+.MailMailboxIntro
+	tx MailMailboxIntroSenderText
+	tx MailMailboxIntroSubjectText
+	tx MailMailboxIntroBody1Text
+	no_card
+	tx MailMailboxIntroBody2Text
+	no_card
+	tx MailMailboxIntroBody3Text
+	db BOOSTER_PRESENT_PACK_1, MAIL_COMMAND_GIVE_BOOSTER
+	end_mail
```

Comment:
So these structures appear to follow this format:

1. Sender text
2. Subject text
3. 0 or more of:
   * Mail text
   * Card command (2 bytes) or $0000
4. $ffff

For the 4 bytes per mail page, I would probably use a specific macro:

```asm
MACRO mail
	tx \1
	IF _NARG > 1
		db \2, \3
	ELSE
		db 0, 0
	ENDC
ENDM
```

And this for example would look like:

```suggestion
.MailMailboxIntro
	tx MailMailboxIntroSenderText
	tx MailMailboxIntroSubjectText
	mail MailMailboxIntroBody1Text
	mail MailMailboxIntroBody2Text
	mail MailMailboxIntroBody3Text, BOOSTER_PRESENT_PACK_1, MAIL_COMMAND_GIVE_BOOSTER
	end_mail
```

Just an idea.